pom.xml

```diff
@@ -280,9 +281,10 @@
             <module>plugins/jdg64</module>
             <module>plugins/jdg65</module>
             <module>plugins/jdg66</module>
+            <module>plugins/jdg70</module>
          </modules>
          <properties>
-            <plugins.jdg>jdg60,jdg61,jdg62,jdg63,jdg64,jdg65,jdg66</plugins.jdg>
+            <plugins.jdg>jdg60,jdg61,jdg62,jdg63,jdg64,jdg65,jdg66,jdg70</plugins.jdg>
```

Comment:
Without public (Beta/GA) build of JDG 7.0, you have to create a profile that triggers jdg70 plugin build. Otherwise, those users who want to build JDG plugins but don't have access to private maven repos would get the build failed.